### DISCUSSION TITLE
Tightening Security --what else can be done?

### DISCUSSION THREAD

**Discussion by lead2crm** [GerryDazoo/Slinger#597] [Q&A]
Is there anything else I need to do to help tighten up the security on this? Is there much to even be worried about?

Presently, i just have Windows Firewall Defender with slingbox marked as an allowed app. My router has it's default firewall software.

The PC that is running this is literally useless beyond this feature and has no cool information or passwords or Networking Connections--not even a printer.

**Comment by sp222222** [comment]
ad naseum - block outbound port 5678 on any and all devices on your network. OUTbound.
